modules/elasticsearch_helper_index_management/elasticsearch_helper_index_management.install

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Implements hook_schema().
+ */
+function elasticsearch_helper_index_management_schema() {
+  $schema['es_reindex_queue'] = [
+    'description' => 'Queue table for elasticsearch re-indexing',
+    'fields' => [
+      'id' => [
+        'description' => 'The primary identifier for product',
+        'type' => 'serial',
+        'not null' => TRUE,
+        'unsigned' => TRUE,
+      ],
+      'plugin_id' => [
+        'description' => 'The index plugin id',
+        'type' => 'varchar',
+        'length' => 255,
```

Comment:
I got this error when installing the module:
`SQL STATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes`
I got this working by reducing the length of varchar tables to 166. Or alternatively change the type from varchar to varchar_ascii.